### DISCUSSION TITLE
user self-defined JSON file

### DISCUSSION THREAD

**Discussion by Sparkle-27** [ConesaLab/SQANTI3#372] [General] [answered]
Hi.  I have thoroughly read through the SQANTI3 wiki documentation, however, I did not find which parameters in the JSON file can be customized and how to set them. For example, is it possible to set "ratio_TSS": 1.5?
 "incomplete-splice_match":[
        {
            "RTS_stage":"FALSE",
            "perc_A_downstream_TTS":[0,59],
            "ratio_TSS":1.5
        }

**Comment by carolinamonzo** [comment]
You can customize any parameters you want from the classification file. You could set ratio_TSS to 1.5, but then it would only remove those that have the ratio equal to it.

**Reply by ChrissiKalk97** [reply]
Is it really the case that only transcripts with ratio_TSS = 1.5 would be removed? Because looking at the SQANTI3 documentation setting a single value for a numeric column should set the lower limit:

https://github.com/ConesaLab/SQANTI3/wiki/Running-SQANTI3-filter#build_json

"Rules can be set for any numeric or character column in the classification file:

Numeric values: in this case, it is possible to define an interval ([X,Y]) or set just the lower limit (X). Please, take into account that the limit values will be included."

It would be great if you can clarify that :)

**Reply by pabloati** [reply, marked as answer]
Hi Chrissi,

If you set the ratio_TSS to 1.5, that will mean that the filter will consider as valid isoforms those that have a ratio_TSS=>1.5
In case you wanted to do the opposite, or for another parameter where you want to take a smaller than interval, just set it from 0 to the desired number, like it is done with the percentage of Adenine downstream of the TTS.

Hope this makes everything clear :)

**Reply by ChrissiKalk97** [reply]
Dear Pablo,

thanks a lot, this is what I thought in the first place. Thanks a lot for confirming :)
